azure-pipelines.yml

```
@@ -31,17 +31,17 @@ jobs:
       arguments: '--configuration $(buildConfiguration) --no-restore --nologo "-p:Version=$(Build.BuildNumber)-development"'
       projects: 'Demo/Demo.csproj'
   - task: DotNetCoreCLI@2
-    displayName: Publish website [net6.0]
+    displayName: Publish website [net8.0]
     inputs:
       command: publish
-      arguments: '--no-build --nologo --configuration $(BuildConfiguration) --output $(build.ArtifactStagingDirectory)/net6.0 /p:PackageVersion=$(Build.BuildNumber)-development --framework net6.0'
+      arguments: '--no-build --nologo --configuration $(BuildConfiguration) --output $(build.ArtifactStagingDirectory)/net8.0 /p:PackageVersion=$(Build.BuildNumber)-development --framework net8.0'
```

```suggestion
      arguments: '--no-build --nologo --configuration $(BuildConfiguration) --output $(build.ArtifactStagingDirectory)/net8.0 /p:PackageVersion=$(Build.BuildNumber)-development'
```

Do we need to specify the target framework here? The demo project only targets one version at a time.